kubespawner/spawner.py

```diff
@@ -109,6 +108,26 @@ def _namespace_default(self):
         """
     ).tag(config=True)
 
+
+    node_selector = Dict(
+        help="""
+        Thes labels used to match the Nodes where Pods will be launched.
+
+        Default is None and means it will be launched in any available Node.
+        
+        For example to match the Nodes that have a label of `disktype: ssd` use:
+            `{"disktype": "ssd"}`
+        """
+    )
+
+    working_dir = Unicode(
+        "/home",
```

Comment:
I think the default should be None, which should make it use whatever is specified in the Docker image (which is the current default). Should probably be called singleuser_working_dir as well (I've been trying to do that for all new properties)